src/systems/camera-system.js

```diff
@@ -207,6 +204,9 @@ export class CameraSystem {
     this.mode = CAMERA_MODE_SCENE_PREVIEW;
     this.snapshot = { audioTransform: new THREE.Matrix4(), matrixWorld: new THREE.Matrix4() };
     this.audioSourceTargetTransform = new THREE.Matrix4();
+    scene.addEventListener("cameraready", ({ detail: { cameraEl } }) => {
+      cameraEl.getObject3D("camera").layers.mask |= LayerMasks.CAMERA_LAYER_VIDEO_TEXTURE_TARGET_MASK;
```

Comment:
Can't you use `.enable(Layers.CAMERA_LAYER_VIDEO_TEXTURE_TARGET)` here?

Reply:
That's true. I'm going to switch to enable and remove the Layers mask enum.